.github/workflows/test.yml

```diff
@@ -63,6 +63,9 @@ jobs:
           ref: ${{ github.head_ref }}
           submodules: "recursive"
 
+      - name: Preserve the ArkanaKeys
```

Comment:
I think we should add a comment for explanation: `Preserve the ArkanaKeys (avoid re-generating new key when refreshing the "sample" project)`